packages/manager/modules/product-offers/README.md

```diff
@@ -71,6 +71,8 @@ export default class {
       getPlanCode: this.getPlanCode.bind(this),
       productName: 'webHosting',
       serviceNameToAddProduct: 'nameOfMyProduct',
+      onPricingSubmit: () => console.log('onPricingSubmit'),
+      onValidateSubmit: () => console.log('onValidateSubmit'),
```

Comment:
```suggestion
```

Reply:
These are examples on the README file of the module. We can let it as is, what do you think ?

Reply:
Nevermind, I didn't notice it was on documentation, my bad